client/my-sites/stats/stats-purchase/index.tsx

```diff
@@ -116,7 +131,7 @@ const StatsPurchasePage = ( { query }: { query: { redirect_uri: string; from: st
 			<div className="stats">
 				<QueryProductsList type="jetpack" />
 				{
-					// TODO: style loading state
+					// TODO: if the page is commercial and already has a commercial plan we can either redirect them or display a message
```

Comment:
FYI the redirection is implemented here: https://github.com/Automattic/wp-calypso/pull/80890/files#diff-7d16e517ddc75ad5e9be5f880600692ed50ecb1e8469c20a43f623fa60b30e27L76

And I agree a message would be better 😃